ts-sequelize/model.ts

```diff
@@ -22,28 +22,44 @@ const KV = sequelize.define('KV', {
     }
   }, {
    
-  });
+});
   
-  (async () => {
+const connect = (async () => {
```

Comment:
I'm not a huge fan of the "connect" promise that you have to remember to await on before using the client. What do you think about bundling that into the client setup instead? For example:
```ts
const sequelize = setupSequelize()

async function setupSequelize() {
  const seq = new Sequelize(`sqlite::memory:`, {logging: false});

  await sequelize.authenticate()

  await sequelize.sync({alter: true})
}

const KV = sequelize.then(client => client.define('KV', { /* ... */}))
```

We'd have to move the annotation to inside the function and track its usage and assignment though.

(Also using `alter` I think that's a migration instead of `force` :: https://sequelize.org/docs/v6/core-concepts/model-basics/#model-synchronization)